src/github-handler/branch-handler.ts

```diff
@@ -71,17 +73,19 @@ async function branch(
   // create branch from primary branch HEAD SHA
   try {
     const baseSHA = await getBranchHead(logger, octokit, origin, baseBranch);
-    await octokit.git.createRef({
-      owner: origin.owner,
-      repo: origin.repo,
-      ref: createRef(name),
-      sha: baseSHA,
-    });
-    logger.info('Successfully created branch');
+    const refData = (
+      await octokit.git.createRef({
+        owner: origin.owner,
+        repo: origin.repo,
+        ref: createRef(name),
+        sha: baseSHA,
+      })
+    ).data;
+    logger.info(`Successfully created branch at ${refData.url}`);
     return baseSHA;
   } catch (err) {
     logger.error('Error when creating branch');
-    throw Error(err.toString());
+    throw Error(err);
```

Comment:
For this one will this work?
```suggestion
    throw err;
```

Reply:
Thanks for revisiting this. I realized this actually does work but not with the current tests.